### PR TITLE
Fix CONT-91: Add support for contributions and donations

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

File: laterpay/__init__.py [hunk 1]
```diff
@@ -302,19 +308,38 @@ def _get_web_url(self,
...
-        Get the URL at which a user can start the checkout process to buy a single item.
+        Get the URL at which a user can start the checkout process to buy a
+        single item or contribute/donate to a campaign.
 
         http://docs.laterpay.net/platform/dialogs/buy/
         """
-        return self._get_web_url(item_definition, 'buy', *args, **kwargs)
+        page_type = 'buy'
+        item_type = item_definition.item_type
+        if not item_type:
```
Comment (kamni) [inline laterpay/__init__.py]: Not a blocker, but this seems a bit odd. What if you omitted this?

**Comment by kamni** [comment]
Nevermind. Comment addressed immediately after I hit enter. :grinning:

File: laterpay/__init__.py (hunk 1 above)
Comment (MarkusH) [reply laterpay/__init__.py]: My first intention was to default to the first if branch for faster execution, but getting coverage right was going to be tricky. Also, premature optimization.

**Comment by MarkusH** [comment]
A well, rebased instead of squashed. Nevermind.

**Pull request merged**